codalab/objects/work_manager.py

```diff
@@ -292,15 +292,8 @@ def update_running_bundle(self, status):
                     print >>sys.stderr, 'Worker.finalize_bundle: installing (copying) dependencies to %s (MakeBundle)' % temp_dir
                     bundle.install_dependencies(self.bundle_store, self.get_parent_dict(bundle), temp_dir, copy=True)
 
-                # Note: uploading will move temp_dir to the bundle store.
-                (data_hash, bundle_store_metadata) = self.bundle_store.upload(sources=[temp_dir],
```

Comment:
Just to confirm: temp_dir is now the actual bundle directory, right?  Probably would be good to note that.